src/main/resources/db/migration/v0.37.0.1__transcript_association_tables.sql

```diff
@@ -0,0 +1,117 @@
+CREATE TABLE transcriptcodingsequenceassociation (
+	id bigint CONSTRAINT transcriptcodingsequenceassociation_pkey PRIMARY KEY,
+	relation_id bigint,
+	transcriptassociationsubject_id bigint,
+	transcriptcodingsequenceassociationobject_id bigint
+);
+
+ALTER TABLE transcriptcodingsequenceassociation ADD CONSTRAINT transcriptcodingsequenceassociation_id_fk
+	FOREIGN KEY (id) REFERENCES evidenceassociation(id);
+ALTER TABLE transcriptcodingsequenceassociation ADD CONSTRAINT transcriptcodingsequenceassociation_relation_id_fk
+	FOREIGN KEY (relation_id) REFERENCES vocabularyterm(id);
+ALTER TABLE transcriptcodingsequenceassociation ADD CONSTRAINT transcriptcodingsequenceassociation_tasubject_id_fk
+	FOREIGN KEY (transcriptassociationsubject_id) REFERENCES transcript(id);
+ALTER TABLE transcriptcodingsequenceassociation ADD CONSTRAINT transcriptcodingsequenceassociation_tcdsaobject_id_fk
+	FOREIGN KEY (transcriptcodingsequenceassociationobject_id) REFERENCES codingsequence(id);
+	
+CREATE INDEX transcriptcdsassociation_relation_index ON transcriptcodingsequenceassociation
+	USING btree (relation_id);
+CREATE INDEX transcriptcdsassociation_subject_index ON transcriptcodingsequenceassociation
+	USING btree (transcriptassociationsubject_id);
+CREATE INDEX transcriptcdsassociation_object_index ON transcriptcodingsequenceassociation
+	USING btree (transcriptcodingsequenceassociationobject_id);
+
+CREATE TABLE transcriptexonassociation (
+	id bigint CONSTRAINT transcriptexonassociation_pkey PRIMARY KEY,
+	relation_id bigint,
+	transcriptassociationsubject_id bigint,
+	transcriptexonassociationobject_id bigint
+);
+
+ALTER TABLE transcriptexonassociation ADD CONSTRAINT transcriptexonassociation_id_fk
+	FOREIGN KEY (id) REFERENCES evidenceassociation(id);
+ALTER TABLE transcriptexonassociation ADD CONSTRAINT transcriptexonassociation_relation_id_fk
+	FOREIGN KEY (relation_id) REFERENCES vocabularyterm(id);
+ALTER TABLE transcriptexonassociation ADD CONSTRAINT transcriptexonassociation_tasubject_id_fk
+	FOREIGN KEY (transcriptassociationsubject_id) REFERENCES transcript(id);
+ALTER TABLE transcriptexonassociation ADD CONSTRAINT transcriptexonassociation_teaobject_id_fk
+	FOREIGN KEY (transcriptexonassociationobject_id) REFERENCES exon(id);
```

Comment:
Maybe I missed something but I see 4 foreign keys and only 3 indexes. I would think there should be 4? Also again further down in this file.

Reply:
4th FK is id field, which is primary key

Reply:
Ahh ok thanks.